selenium_tests/class_roster/class_roster_tests.py

```diff
@@ -0,0 +1,25 @@
+
+from selenium_tests.class_roster.class_roster_base_test_case \
+    import ClassRosterBaseTestCase
+
+
+
+class ClassRosterTests(ClassRosterBaseTestCase):
+
+    def test_class_roster_url_link(self):
+
+        """
+        TLT-1767:  This test checks to make sure that the class roster URL
+        contains values that matches the expected values
+        AC #6, Test Case #9
+        """
+        course_name_display = self.test_settings['roster_text_display']
+        url_link_course_number = self.test_settings['url_link_course_number']
+
+        # Get the link URL matching the course name display
+        link_url = self.main_page.get_link_url(course_name_display)
+
+        # Verify that the expect course number appears in the link URL
+        self.assertTrue(url_link_course_number in link_url,
```

Comment:
Great idea.  I thought I start off with course_number first.  I'll add in a check for the term as well, since that is tied in with the course.  For the robustness of the test, I'll skip the user_id check if/when the test user gets changed.